modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/CacheInfo.java

```diff
@@ -381,10 +391,28 @@ public Object name(VisorViewCacheCmd cmd) {
         affinityClsName = U.readString(in);
         cachesCnt = in.readInt();
         atomicityMode = protoVer >= V2 ? CacheAtomicityMode.fromOrdinal(in.readByte()) : null;
+        heapEntriesCount = protoVer >= V5 ? (Long)in.readObject() : null;
+        offHeapPrimaryEntriesCount = protoVer >= V5 ? (Long)in.readObject() : null;
     }
 
     /** {@inheritDoc} */
     @Override public String toString() {
         return S.toString(CacheInfo.class, this);
     }
+
+    public void setHeapEntriesCount(Long heapEntriesCount) {
+        this.heapEntriesCount = heapEntriesCount;
+    }
+
+    public Long getHeapEntriesCount() {
+        return heapEntriesCount;
+    }
+
+    public void setOffHeapPrimaryEntriesCount(Long offHeapPrimaryEntriesCount) {
```

Comment:
description required.

Reply:
@param is not enough for description, plz add desc.

Reply:
Fixed